project/npda/templates/npda/visit_form.html

```diff
@@ -40,310 +40,10 @@
-                          </div>
-                          <div class="flex justify-end w-full md:w-2/3 flex-wrap">
-                            {% if field.field.widget|is_select %}
-                              <select id="{{ field.id_for_label }}"
-                                      name="{{ field.html_name }}"
-                                      class="select rcpch-select rounded-none {% if not can_alter_this_audit_year_submission or not can_complete_questionnaire %}opacity-40 pointer-events-none{% endif %}">
-                                {% for choice in field.field.choices %}
-                                  <option value="{{ choice.0 }}"
-                                          {% if field.value|stringformat:'s' == choice.0|stringformat:'s' %} selected="{{ field.value }}" {% endif %}>
-                                    {{ choice.1 }}
-                                  </option>
-                                {% endfor %}
-                              </select>
-                            {% elif field.field.widget|is_dateinput %}
-                              <input type="date" id="{{ field.id_for_label }}" name="{{ field.html_name }}" class="input rcpch-input-text flex-grow basis-2/3 {% if not can_alter_this_audit_year_submission or not can_complete_questionnaire %}opacity-40 pointer-events-none{% endif %}" {% if field.value %}value={{ field.value|stringformat:'s' }}{% endif %}>
-                              <div class="basis-1/3 flex">
-                                <button type='button'
-                                        _="on click set #{{ field.id_for_label }}'s value to '{% today_date %}'"
-                                        class="btn rcpch-btn bg-rcpch_light_blue border border-rcpch_light_blue hover:bg-rcpch_strong_blue hover:border-rcpch_strong_blue {% if not can_alter_this_audit_year_submission or not can_complete_questionnaire %}opacity-40 pointer-events-none{% endif %}">
-                                  Today
-                                </button>
-                                <button type='button'
-                                        _="on click set #{{ field.id_for_label }}'s value to #id_visit_date.value"
-                                        class="btn rcpch-btn bg-rcpch_light_blue border border-rcpch_light_blue hover:bg-rcpch_strong_blue hover:border-rcpch_strong_blue {% if not can_alter_this_audit_year_submission or not can_complete_questionnaire %}opacity-40 pointer-events-none{% endif %}">
-                                  Fill with Visit Date
-                                </button>
-                              </div>
-                            {% else %}
-                              <!-- these are the field inputs: do not want to show centile/sds  -->
-                              {% is_not_excluded_centile_field field as included_field %}
-                              {% if included_field or field.id_for_label == 'id_bmi' %}
```

Comment:
This code never ran as we maintain an include list of fields in VisitForm. I've raised https://github.com/rcpch/national-paediatric-diabetes-audit/issues/501 to put it back